examples/torch/classification/main.py

```diff
@@ -157,8 +158,8 @@ def autoq_eval_fn(model, eval_loader):
 
     model.to(config.device)
 
-    resuming_model_sd, resuming_checkpoint = load_resuming_checkpoint(resuming_checkpoint_path)
-    compression_ctrl, model = create_compressed_model(model, nncf_config, resuming_state_dict=resuming_model_sd)
+    nncf_checkpoint, resuming_checkpoint = load_resuming_checkpoint(resuming_checkpoint_path)
+    compression_ctrl, model = create_compressed_model(model, nncf_config, compression_state=nncf_checkpoint)
```

Comment:
`compression_state` parameter in fact contains not only values pertaining to the compression state, but also the underlying FP32 weights as well. Perhaps it could be renamed to `compressed_checkpoint` or even to `nncf_checkpoint`? Same with `get_compression_state`

Reply:
As far as I understood, most probably there're some technical issues with saving model state within nncf-checkpoint in TF, instead there's a separate checkpoint for model. I thought `checkpoint` in the name can be confusing.

@alexsu52 @andrey-churkin @daniil-lyakhov any objections for renaming `compression_state` to `nncf_checkpoint`?

Reply:
The corresponding parameter of `create_compressed_model` must have a single name and a single meaning across framework. For me `(nncf_checkpoint | compressed_checkpoint )= framework_specific_fp32_only_model_data + compression_state`. I think we should preserve this meaning everywhere. If TF's create_compressed_model accepts a `compression_state`, while PT's create_compressed_model accepts an `(nncf_checkpoint | compressed_checkpoint )`, then there is no other way than to have different signatures for the functions and document this difference carefully.

Reply:
kept CompressionState only, that contains builder and ctrl states